docker/ubuntu18-ci/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM ubuntu:18.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
+# Update Ubuntu Software repository
+RUN apt update && \
+    apt-get install -y \
+      git autoconf automake libtool make libreadline-dev texinfo \
+      pkg-config libpam0g-dev libjson-c-dev bison flex python3-pytest \
+      libc-ares-dev python3-dev libsystemd-dev python-ipaddress python3-sphinx \
+      install-info build-essential libsystemd-dev libsnmp-dev perl libcap-dev \
+      libelf-dev \
+      sudo gdb iputils-ping time \
+      mininet python-pip iproute2 iperf && \
+      pip install ipaddr && \
+      pip install "pytest<5" && \
+      pip install "scapy>=2.4.2" && \
```

Comment:
nit: I would suggest using requirements.txt in such cases because it will be a mess long-term with lots of Dockerfiles.